docs/visualize/tilemap.asciidoc

```diff
@@ -1,47 +1,136 @@
+placed in the center of all documents in the bucket, and a more accurate visualization is created. When deselected, the markers are placed in the center
 of the geohash grid cell.
 
+NOTE: When you have multiple values in the geo_point, the coordinate map is unable to accurately calculate the geo_centroid.
+
 [float]
-[[navigate-map]]
-=== Navigate the coordinate map
+[[navigate-coordinate-map]]
+==== Navigate the coordinate map
 
-Use the following navigation options:
+To navigate the coordinate map, use the following navigation options:
 
 * To move the map center, click and hold anywhere on the map and move the cursor.
+
 * To change the zoom level, click *Zoom In* or *Zoom out* image:images/viz-zoom.png[].
+
 * To automatically crop the map boundaries to the
 geohash buckets that have at least one result, click *Fit Data Bounds* image:images/viz-fit-bounds.png[].
+
+[float]
+[[region-map]]
+=== Region map
+
+Use region maps when you want to show statistical data on a geographic area, such as a county, country, province, or state. For example, use a region map if you want to see the average sales for each country with the sample eCommerce order data.
+
+[role="screenshot"]
+image::images/visualize_region_map_example.png[]
+
+[float]
+[[build-region-maps]]
+==== Build a region map
+
+. In `kibana.yml`, set `xpack.maps.showMapVisualizationTypes` to `true`.
+
+. To display your data on the region map, use the following aggregations:
+
+* <<visualize-metric-aggregations,Metric>>
+* <<visualize-sibling-pipeline-aggregations,Sibling pipeline>>
+* <<visualize-bucket-aggregations,Terms bucket aggregation>>
+
+[float]
+[[navigate-region-map]]
+==== Navigate the region map
+
+To navigate the region map, use the following options:
+
+* To change the zoom level, click *Zoom In* or *Zoom out* image:images/viz-zoom.png[].
+
+* To automatically crop the map boundaries, click *Fit Data Bounds* image:images/viz-fit-bounds.png[].
+
+[float]
+[[heat-map]]
+=== Heat map
+
+Use heat maps when your data set includes categorical data. For example, use a heat map to see the flights of origin countries compared to destination countries using the sample flight data.
+
+[role="screenshot"]
+image::images/visualize_heat_map_example.png[]
+
+[float]
+[[build-heat-map]]
+==== Build a heat map
+
+To display your data on the heat map, use the following aggregations:
+
+* <<visualize-metric-aggregations,Metric>>
+* <<visualize-parent-pipeline-aggregations,Parent pipeline>>
+* <<visualize-sibling-pipeline-aggregations,Sibling pipeline>>
+* <<visualize-bucket-aggregations,Bucket>>
+
+[float]
+[[navigate-heatmap]]
+==== Change the color ranges
+
+When only one color displays on the heat map, you might need to change the color ranges.
+
+To specify the number of color ranges:
+
+. Click *Options*.
+
+. Enter the *Number of colors* to display.
+
+To specify custom ranges:
+
+. Click *Options*.
+
+. Select *Use custom ranges*.
+
+. Enter the ranges to display.
```

Comment:
If the examples you mention aren't in the sample data sets, we should think about creating examples for the user to follow and also include tips and tricks, if any.